pytuya/__init__.py

```diff
@@ -374,7 +405,17 @@ def set_timer(self, num_secs):
         devices = status['dps']
         devices_numbers = list(devices.keys())
         devices_numbers.sort()
-        dps_id = devices_numbers[-1]
+        print(devices_numbers)
+        dps_list = []
+        for dps in devices_numbers:
+            if int(dps) > 5:
+                dps_list.append(int(dps))
+        dps_list.sort()
+
+        if number_dps >= 1 and number_dps <=4:
+            number_dps = number_dps-1
+
```

Comment:
I'm not entirely sure what this is doing - it needs an explanation

Reply:
Hum if for detect the plug number > 4 but the return true or false or numerical I check for fix last time